modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ItemActionForm.tsx

```diff
@@ -206,6 +206,10 @@ const ItemActionForm = ({
 		getIcons();
 	}, [spritemap]);
 
+	useEffect(() => {
+		validateForm();
+	});
+
```

Comment:
This kinda looks a weird pattern to me but I understand its purpose. Perhaps we may just validate if `!editing` or `initialValues` comes empty.

Anyways I'm more than ok with sending this as it is

Reply:
I don't like this. It will technically work, but I'd call it an anti-pattern. There are many scenario in which component can re-render without changing form fields with validation, so this has potential to severely affect performance. And that's on top of typical small performance hit and bugs potential that any `useEffect` causes. It is also a global effect, so it is easy to miss when trying to figure out code and debug.

Even ignoring performance and readability, this should be unneccesary. We already validate on each relevant input blur, and on save action. What case does this intend to cover? If it's to immediately invalidate form, this changes intentional behavior where form _looks_ valid initially. Errors intentionally appear only on first save, or blur.

Reply:
It should be the other way round, validate when `!!editing` or receiving `initialValues`, otherwise the form will be empty and invalid.